src/framework/piral-core/src/helpers.test.tsx

```diff
@@ -191,10 +191,10 @@ describe('Piral-Core helpers module', () => {
   });
 
   it('createPiletOptions runs in PILET_DEBUG context', () => {
-    const wasUndefined = process.env.DEBUG_PILET === undefined;
+    const wasUndefined = window['dbg:pilet-api'] === undefined;
 
     // Arrange
-    process.env.DEBUG_PILET = 'localhost:1234';
+    window['dbg:pilet-api'] = 'localhost:1234';
```

Comment:
Not sure why the original value of this was `localhost:1234`, shouldn't it be `/$pilet-api`?

Reply:
Yeah I think that makes more sense!

Not sure here; always depends what should be tested; I guess here it was just about having some string in there.